x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/esql/esql.ts

```diff
@@ -225,6 +226,16 @@ export const esqlExecutor = async ({
           break;
         }
       }
+      if (
+        completeRule.ruleParams.responseActions?.length &&
+        result.createdSignalsCount &&
+        scheduleNotificationResponseActionsService
+      ) {
+        scheduleNotificationResponseActionsService({
```

Comment:
Do we need to `await` here? Also, I think it would be beneficial to see unit and integration tests exemplifying the expected behavior when this function exists / throws an error within each rule type executor.

Reply:
Good idea - but we do not want to await this, we in no circumstance would like to block the rule execution - response actions live their own life ;p 
I'll try to add more tests though 👍